### DISCUSSION TITLE
NBMiner v39.3

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#449] [General]
discus here.

**Comment by PJTraut** [comment]
Mining ETH with 6x 3080LHR SEAHAWKS on HiveOS/Hiveon:
- lhr 66.8 
- core 1080
- mem 2500
- pl 235
>>stable 400-401 MH/s<<
ambient heat: 21C
fan: 40%
cards: 35-40C
reported power fluctuates 905-1100w

The MUtl is 56-65 and spikes to 100 (assume it is the "magic sauce" kicking in).
**When I drop the core below 1080, there seems that the cards run higher MUtl and hit "100" more often, and the reported power spikes over 1200 sometimes.**

Great job to the team - compared to 39.2, dropped 6MH/s on the rig for about 25% power saving and 8C less heat per card :)

**Comment by nowwheree** [comment]
great job guys im getting 54 mhs on my 3070 ti founder edition lhr

**Reply by Vance-Organics** [reply]
Yes thank you! 83mhs on my 3080Ti and I haven't even tweaked it yet

**Comment by 630472521** [comment]
why NBminer 39.5 is mining ,but i cannot see my account in pool? i use F2pool
but i use other miner tools ------ Bminer-v16.4.7-  i can see my account in F2pool . i don not why 
and hope someone can tell me how adjust  " LHR " mode  in  Bminer-v16.4.7

**Comment by bestman12122** [comment]
How does this work

**Comment by smithdawn24admin-star** [comment]
Hello super excted to get started in brand new but learn very fast  see yall in the blocks
